web/models.py

```
@@ -2864,6 +2864,96 @@ def __str__(self):
+    """
+
+    id = models.AutoField(primary_key=True)
+    student = models.ForeignKey(User, on_delete=models.CASCADE, related_name="session_feedback")
+    session = models.ForeignKey(Session, on_delete=models.CASCADE, related_name="feedback")
+    rose = models.TextField(blank=True, help_text="Something that went well / they liked")
+    bud = models.TextField(blank=True, help_text="Something they want to learn more about or that has potential")
+    thorn = models.TextField(blank=True, help_text="Something that was difficult, confusing, or frustrating")
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        unique_together = ["student", "session"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)